.github/workflows/tests.yml

```diff
@@ -152,7 +163,10 @@ jobs:
           CHANGED_EXAMPLES=$(echo "$CHANGED" | grep examples )
           set -e
           ALL_EXAMPLES=$(ls -1d label_studio_ml/examples/*/ | cut -d'/' -f3- | cut -d'/' -f1)
-          if [[ $(echo "$CHANGED" | grep -c -v ^$) == $(echo "$CHANGED_EXAMPLES" | grep -c -v ^$) ]]; then
+          if [[ $GITHUB_REF == 'refs/heads/master' || $GITHUB_REF =~ ^refs/heads/release/.* ]]; then
+            echo "All in matrix because we are on master or a release branch"
+            echo "matrix={\"ml_backend\":[$(echo "$ALL_EXAMPLES" | xargs | sed -e 's/ /\",\"/g' -e 's/^/\"/' -e 's/$/\"/')]}" >> $GITHUB_OUTPUT
+          elif [[ $(echo "$CHANGED" | grep -c -v ^$) == $(echo "$CHANGED_EXAMPLES" | grep -c -v ^$) ]]; then
             echo "Finding examples to test"
```

Comment:
This should work work to trigger all tests when running from master or release branches. Impossible to test the names are correct until we merge.  I did test actual execution by adding the PR branch name as well and all tests were run in that case so we do know the code executes properly.